snuba/query/functions.py

```diff
@@ -0,0 +1,104 @@
+# This is supposed to enumerate the functions snuba supports (with their
```

Comment:
On the TODO list we have a task to introduce a list of allowed functions the client can use when querying snuba. It is not yet fully done but it is supposed to show up soon (as we do not have control on functions called by the client now).
The list of valid aggregation functions should be part of such list and stored in the same place, so we will be certain the list of aggregation functions is always valid.
If the list of aggregation functions is physically a subset of the valid functions, there is no way clickhouse can introduce a new aggregation function without us noticing because such function will be rejected by snuba.